src/resources/styles.ts

```diff
@@ -142,11 +142,10 @@ export const haStyleDialog = css`
     --mdc-dialog-max-width: 600px;
     --mdc-dialog-max-width: min(600px, 95vw);
     --justify-action-buttons: space-between;
-    --dialog-container-padding: var(--safe-area-inset-top, var(--ha-space-0))
-      var(--safe-area-inset-right, var(--ha-space-0))
-      var(--safe-area-inset-bottom, var(--ha-space-0))
-      var(--safe-area-inset-left, var(--ha-space-0));
-    --dialog-surface-padding: var(--ha-space-0);
+    --dialog-container-padding: var(--safe-area-inset-top, 0)
+      var(--safe-area-inset-right, 0) var(--safe-area-inset-bottom, 0)
+      var(--safe-area-inset-left, 0);
+    --dialog-surface-padding: 0;
```

Comment:
```
    --dialog-container-padding: var(--safe-area-inset-top, 0px)
      var(--safe-area-inset-right, 0px) var(--safe-area-inset-bottom, 0px)
      var(--safe-area-inset-left, 0px);
    --dialog-surface-padding: 0px;
```

Last one in case used in calculations elsewhere